### PR TITLE
[REL] Migration sale_stock_ux to 12.0

### PR DESCRIPTION
**Pull request by nicomacr**

### CODE REVIEW

File: sale_stock_ux/models/sale_order_line.py [hunk 1]
```diff
@@ -23,8 +23,8 @@ class SaleOrderLine(models.Model):
     # TODO This should be computed field
```
Comment (zaoral) [inline sale_stock_ux/models/sale_order_line.py]: remove comment, you already change the field to computed field

File: sale_stock_ux/models/sale_order_line.py [hunk 2]
```diff
@@ -23,8 +23,8 @@ class SaleOrderLine(models.Model):
...
     qty_returned = fields.Float(
         string='Returned',
+        compute='_compute_qty_returned',
         copy=False,
-        default=0.0,
         digits=dp.get_precision('Product Unit of Measure'),
         readonly=True,
```
Comment (zaoral) [inline sale_stock_ux/models/sale_order_line.py]: I think that this readonly is not necessary since now is a compute fieñd

**Review by zaoral — COMMENTED** [review]
LGTM +1 @nicomacr 

I leave you here some comments, you decide if you will like to apply it:

* I think by that maybe the field all_qty_delivered could be removed
* remove string='Delivery Status',  of delivery_field in the sale.order.line already same as name of the field
* rename file stock_return_picking_line_views.xml to  stock_return_picking_views.xml

Best Regards

**Comment by zaoral** [comment]
note: I was not able to make functional test, runbot is failling because the product_ux module is missing (maybe has not been migrated yet)

Please if you could make a rebase will be great

**Comment by nicomacr** [comment]
@zaoral the changes suggested has been apply

**Review by zaoral — APPROVED** [review]
Excellent! @nicomacr 

There is a yellow runbot, it seems like is because of multiple warnings, ones related to this and other modules.

Please could you review if it is possible to fix them?
Best Regards

**Pull request merged**